TRACK_README.md

```diff
@@ -30,6 +49,14 @@ Please keep the following in mind:
 
 - Watch out for trailing spaces, extra blank lines, and spaces in blank lines.
 
+- All the tests for {{LANGUAGE}} exercises can be run from the top level of the repo with ... Please run this command before submitting your PR.
+
+## Contributing a New Exercise ##
+
+- All Exercism exercises must be defined in [x-common](https://github.com/exercism/x-common/tree/master/exercises) before they are implemented for a specific track. Please submit a PR there if your exercise is new to Exercism.
```

Comment:
Whoever is documenting https://github.com/exercism/trackler/pull/24, please modify this as necessary.